### PR TITLE
Beancount 3 compatibility

### PR DESCRIPTION
**Pull request by lvh**
I haven't tested this yet but, refs (closes?) #4.

### CODE REVIEW

File: beancount_periodic/common/config.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
 
 from dateutil.relativedelta import relativedelta
 
-from beancount.utils.date_utils import parse_date_liberally
+try:
+    from beancount.utils.date_utils import parse_date_liberally
+except ImportError:
+    from beangulp.date_utils import parse_date_liberally
```
Comment (autrilla) [inline beancount_periodic/common/config.py:13]:
```suggestion
    from beanprice.date_utils import parse_date_liberally
```

(also, hi, long time not see :)

Comment (autrilla) [reply beancount_periodic/common/config.py:13]: I've tested this works with https://github.com/autrilla/beancount-periodic/

Comment (lvh) [reply beancount_periodic/common/config.py:13]: Howdy! Long time no see indeed!

**Comment by lvh** [comment]
@dallaslu Hi! Any chance this could be reviewed/merged? I'd prefer to avoid a fork. Thanks in advance :)

**Pull request merged**